src/Promact.TestCaseManagement.DomainModel/Models/Module/Module.cs

```diff
@@ -6,13 +6,13 @@ namespace Promact.TestCaseManagement.DomainModel.Models.Module
 {
     public class Module : TestCaseManagementBase
     {
-        #region Public Properties
-
         [Required]
         public string Name { get; set; }
 
-        public virtual ICollection<ModuleTestCaseMapping> ModelTestCaseMapping { get; set; }
+        public int ProjectId { get; set; }
+
+        public virtual Project.Project Project { get; set; }
```

Comment:
Don't nest classes under module names. i.e, keep all class files under Modals only. No need to separate them